src/utils/index.ts

```diff
@@ -19,6 +19,9 @@ import { ServiceAccountCredential, ComputeEngineCredential } from '../auth/crede
 
 import * as validator from './validator';
 
+const { version } = require('../../package.json'); // eslint-disable-line @typescript-eslint/no-var-requires
```

Comment:
Let's expose this as a function. That's more future proof, and also defers loading the `package.json` until absolutely necessary.

```
let sdkVersion: string;

export function getSdkVersion() {
  if (!sdkVersion) {
    // load from file
  }

  return sdkVersion;
}
```